src/pyFAI/gui/pilx/MainWindow.py

```diff
@@ -180,12 +194,12 @@ def displayImageAtIndices(self, indices: ImageIndices):
         col = indices.col
 
         with h5py.File(self._file_name, "r") as h5file:
-            map_shape = get_dataset(h5file, "/entry_0000/pyFAI/result/intensity").shape
+            map_shape = get_dataset(h5file, f"{self._nxprocess_path}/result/intensity").shape
```

Comment:
Same as before: you can put `nxprocess` in a variable to reuse it below:

```suggestion
            nxprocess = h5file[self._nxprocess_path]
            map_shape = get_dataset(nxprocess, "/result/intensity").shape
```